eng/packaging.targets

```diff
@@ -26,4 +26,16 @@
                       '$(_excludeCompile)' == 'true' and
                       '%(Dependency.Identity)' != '_._'" />
   </Target>
+
+  <PropertyGroup>
+    <PkgProjPath>$(MSBuildProjectDirectory)\..\pkg\$(MSBuildProjectName).pkgproj</PkgProjPath>
+    <!-- Point to our cusotm pack target which builds the pkgproj instead. -->
+    <PackDependsOn Condition="'$(IsSourceProject)' == 'true' and
+                              Exists('$(PkgProjPath)')">_BuildPkgProj</PackDependsOn>
+  </PropertyGroup>
+
+  <Target Name="_BuildPkgProj">
+    <MSBuild Projects="$(PkgProjPath)"
+             Targets="Build" />
```

Comment:
Nit: you can omit Targets.

Reply:
I intentionally put it there as I wasn't sure what the default target for a pkgproj is. If there isn't one specified, MSBuild always picks the first target found.